eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionReportWriter.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.pipeline.io;
+
+import uk.ac.ebi.eva.accession.core.ISubmittedVariant;
+import uk.ac.ebi.eva.accession.core.SubmittedVariant;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class AccessionReportWriter {
+
+    private static final String VCF_MISSING_VALUE = ".";
+
+    private FastaSequenceReader fastaSequenceReader;
+
+    private boolean headerWritten;
+
+    private final BufferedWriter fileWriter;
+
+    private String accessionPrefix;
+
+    public AccessionReportWriter(File output, FastaSequenceReader fastaSequenceReader) throws IOException {
+        this.fastaSequenceReader = fastaSequenceReader;
+        this.headerWritten = false;
+        this.fileWriter = new BufferedWriter(new FileWriter(output));
+        this.accessionPrefix = "ss";
+    }
+
+    public String getAccessionPrefix() {
+        return accessionPrefix;
+    }
+
+    public void setAccessionPrefix(String accessionPrefix) {
+        this.accessionPrefix = accessionPrefix;
+    }
+
+    public void write(Map<Long, ISubmittedVariant> accessions) throws IOException {
+        if (!headerWritten) {
```

Comment:
This could cause a problem. The first time it will write the header but in case the program is restarted and the Bean needs to be created again the attribute headerWritten will be set to false (constructor) but the header will be written again even though it was written previously in the file.

Reply:
It could be possible to store it in the job execution context.

Reply:
The file is overwritten, not appended

Reply:
after some discussion, this was changed to append to the file and write in the execution context if the header was already written.